docs/limitations.md

```diff
@@ -8,7 +8,9 @@ enterprise: 'yes'
 *   DC/OS Spark only supports submitting jars and Python scripts. It
 does not support R.
 
-*   Mesosphere does not provide support for Spark app development.
+*   Mesosphere does not provide support for Spark app development,
```

Comment:
I think we should just merge, and let @sascala make edits afterwards.  It's probably easier for her to edit herself than make a bunch of PR comments.